tasks/bump.js

```diff
@@ -77,14 +82,12 @@ module.exports = function(grunt) {
 
     var globalVersion; // when bumping multiple files
     var gitVersion;    // when bumping using `git describe`
-    var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)([\d||A-a|.|-]*)([\'|\"]?)/i;
-
+    var VERSION_REGEXP = new RegExp('([\\\'|\\\"]?version[\\\'|\\\"]?[ ]*:[ ]*[\\\'|\\\"]?)(\\\d+\\\.\\\d+\\\.\\\d+(-'+opts.prereleaseName+'\\\.\\\d+)?(-\\\d+)?)[\\\d||A-a|.|-]*([\\\'|\\\"]?)', 'i');
```

Comment:
didnt notice this before, but shouldn't the regex replace:

`(-'+opts.prereleaseName+'\\\.\\\d+)?`

with something that won't translate to: `(-null\\\.\\\d+)?` like:

`(-'+ (opts.prereleaseName || '') +'\\\.\\\d+)?`